release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -17,7 +17,7 @@ The release manager must:
 Whenever a new Kubernetes minor version is released, our kubernetes-csi CI jobs
 must be updated.
 
-[Our CI jobs](https://k8s-testgrid.appspot.com/sig-storage-csi-ci) have the
```

Comment:
@abhijeetgauravm.  release-tools is a subtree. The main code can be found here https://github.com/kubernetes-csi/csi-release-tools.

Can you please revert your changes in this (`release-tools/SIDECAR_RELEASE_PROCESS.md`) file? This will be updated automatically.

Reply:
ping @abhijeetgauravm

Reply:
ok sure @letmerecall